iconservice/deploy/__init__.py

```diff
@@ -24,4 +24,3 @@ class DeployType(IntEnum):
 class DeployState(IntEnum):
     INACTIVE = 0
     ACTIVE = 1
-
```

Comment:
should revert

Reply:
An empty line was added to the bottom to comply with pep8.